### PR TITLE
Fix minor grammar issues

### PR DESCRIPTION
**Pull request by stubbybubby**
Inserted apostrophe's' when necessary and fixed the placement of a few conjunctions.

### CODE REVIEW

File: basics_dev/style-guide.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Currently Qubes OS is using the following fonts for our website, branding, and o
 
 ## Colors
 
-The following **grayscale** colors are currently used on the Qubes website, documentation, and will eventually match colors within the OS itself.
+The following **grayscale** colors are currently used on the Qubes website and documentation, and will eventually match colors within the OS itself.
```
Comment (andrewdavidwong) [inline basics_dev/style-guide.md]: This is grammatically incorrect.

File: basics_dev/style-guide.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ The following **grayscale** colors are currently used on the Qubes website, docu
 {% endfor %}
 </div>
 
-The following **colors** are currently being used on the Qubes website, documentation, and will eventually match the colors within the OS itself!
+The following **colors** are currently being used on the Qubes website and documentation, and will eventually match the colors within the OS itself!
```
Comment (andrewdavidwong) [inline basics_dev/style-guide.md]: This is grammatically incorrect.

**Comment by stubbybubby** [comment]
Are you sure those sections are grammatically incorrect? 

"the Qubes website and documentation" in the preposition "on the Qubes website and documentation" is a compound object. The sentence "The following colors are currently being used on the Qubes website, documentation,..." doesn't make sense. That or you're talking about some English rule regarding multiple objects in a preposition that I'm unaware of.

**Comment by andrewdavidwong** [comment]
> "the Qubes website and documentation" in the preposition "on the Qubes website and documentation" is a compound object. There should be no comma there, not unless it's a list of objects (which it isn't because there's only two nouns). The clause "The following colors are currently being used on the Qubes website, documentation,..." doesn't make sense. That or you're talking about some English rule regarding multiple objects in a preposition that I'm unaware of.

The above is true, but the result of your edit is still grammatically incorrect, because "will eventually match the colors within the OS itself" isn't an independent clause. (This is a very minor issue, but then again so is the one you're trying to correct.)

**Comment by stubbybubby** [comment]
https://puu.sh/v9wK8/644d281bec.jpg

**Pull request merged**